docs/Formatting.md

```diff
@@ -6,29 +6,15 @@
 
 ***
 
-### xml.format.enforceQuoteStyle
-
-Enforce `preferred` quote style (set by `xml.preferences.quoteStyle`) or `ignore` quote style when formatting.
-
-For instance, when set to `preferred` with `xml.preferences.quoteStyle` set to `single`, the following document:
-
-  ```xml
-  <?xml version="1.0" encoding="UTF-8"?>
-  <root>
-    <child attribute='value' otherAttribute="otherValue"></child>
-  </root>
-  ```
-
-will be formatted to:
-
-  ```xml
-  <?xml version='1.0' encoding='UTF-8'?>
-  <root>
-    <child attribute='value' otherAttribute='otherValue'></child>
-  </root>
-  ```
+### xml.format.experimental
 
-No changes to quotes will occur during formatting if `xml.format.enforceQuoteStyle` is set to `ignore`.
+  As we have more and more issues with our current XML formatter, we decide to redo our formatter to try to fix them. This new formatter is in experimental state because it doesn't support all current formatter settings and it could have some bugs. To enable this experimental formatter you can set to true the settings xml.format.experimental. Once we will have enough good feedback,  and we will support all current formatting settings, we will replace the current formatter with the experimental formatter. Don't hesitate to create issues to give us feedback with this experimental formatter.
+  
+  The current formatter uses the DOM document and rewrite the XML document or a fragment of XML (when range formatting is processed). This strategy provides a lot of bugs if XML is not valid (ex : <% will format with <null).
+  
+  The new strategy is to format the current XML by adding or removing some spaces without updating the XML content. The experimental formatter follows the same behavior than Oxygen XML. It categorizes each element as mixed, etc. You can use too `xml:space="preserve"` to preserve spaces in some elements or use `xml.format.preserveSpace` to add a given tag element which must preserve spaces.
+  
+  Set to `true` to enable experimental formatter. Defaults to `false`.
```

Comment:
Maybe put this as the first line underneath the title?

Reply:
fixed